### DISCUSSION TITLE
Prebuild in storage even without the existence of a single codespace

### DISCUSSION THREAD

**Discussion by SohamBorate** [community/community#140097] [Codespaces] [answered]
### Select Topic Area

Question

### Body

It appears that i have been using codespaces prebuild storage yet i cant find the repository where any codespace is active
![image_2024-09-29_162926321](https://github.com/user-attachments/assets/513191fc-cd1c-466d-97a0-0698a208e40a)
![image_2024-09-29_163008910](https://github.com/user-attachments/assets/7f58172d-929c-4a26-91ad-d09a33383417)

**Comment by SohamBorate** [comment, marked as answer]
Found where the prebuild was stored, it was in a repository where I had started codespaces and then stopped after sometime (this happened a few months ago).

**Reply by Monsieur-Nico** [reply]
How can I find out which repository has codespaces enabled?

**Reply by SohamBorate** [reply]
> How can I find out which repository has codespaces enabled?

https://github.com/codespaces

Note: This does not show where prebuilds are stored. You would have to check the settings of each and every repository to find where the prebuilds are stored.

**Comment by ChrisZaikis** [comment]
https://github.com/codespaces
